bcda/responseutils/writer.go

```diff
@@ -117,10 +117,26 @@ func CreateCapabilityStatement(reldate time.Time, relversion, baseurl string) *f
 			},
 		},
 	}
-
+	checkEnv(statement, baseurl)
 	return statement
 }
 
+func checkEnv(statement *fhirmodels.CapabilityStatement, baseUrl string) {
+	if os.Getenv("ENABLE_PATIENT_EXPORT") == "true" {
+		restComponent := statement.Rest[0].Operation
+
+		element := fhirmodels.CapabilityStatementRestOperationComponent{
+			Name: "export",
```

Comment:
Is the `export` name defined by FHIR spec? Wondering if it would make sense to be more specific so they are different, e.g., `patient-export` and `eob-export`.

Reply:
Yeah good question. I searched the fhir spec and used this as a reference: "The name of the operation or query. For an operation, this is the name prefixed with $ and used in the URL"

I agree with your idea, I just don't know how flexible we can be here and still follow the standard.

Reply:
Aha, that's what I was looking for. Thanks for the info!